<!DOCTYPE html>
<html>
<head>
	<title>Poster Index</title>
	<style type="text/css">
		html {
			font-family: sans-serif;
		}
		.main {
			margin: 1cm;
			font-size: 120%;
		}
		td {
			padding-bottom: .5ex;
		}
		th {
			text-align: left;
			background-color: #aaaaaa;
		}
	</style>
</head>
<body>
	<div class="main">
<table>
	<tr><th>Poster number</th> <th>Presenter</th> <th>Poster title</th></tr>
<tr><td>A1</td> <td>Aditya Chawla</td> <td>Deep Mutational Scanning of SERPINA1 Gene For Proactive Diagnosis of Alpha-1 Antitrypsin Deficiency</td></tr>
<tr><td>A2</td> <td>Alistair Dunham</td> <td>Exploring amino acid functions in a deep mutational landscape</td></tr>
<tr><td>A3</td> <td>Ammar Tareen</td> <td>MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect</td></tr>
<tr><td>A4</td> <td>Andrew Waters</td> <td>Variant Library Annotation Tool (VaLiAnT): an oligonucleotide library design and annotation tool for Saturation Genome Editing and other Deep Mutational Scanning experiments</td></tr>
<tr><td>A5</td> <td>Anna Axakova</td> <td>Towards a Proactive Missense Variant Effect Map of the Autoimmune Regulator Gene AIRE</td></tr>
<tr><td>A6</td> <td>Anna Posfai</td> <td>Gauge freedoms and the geometry of sequence-function relationships</td></tr>
<tr><td>A7</td> <td>Ashley Hall</td> <td>Measurement and phenotypic consequences of ribosomal DNA copy number variation</td></tr>
<tr><td>A8</td> <td>Berk Alpay</td> <td>A Probabilistic Model of RNA Constraints on Viral Evolution</td></tr>
<tr><td>A9</td> <td>Caelan Radford</td> <td>Investigating patterns of epistasis in HIV Envelope using deep mutational scanning of combinations of mutations</td></tr>
<tr><td>A10</td> <td>Cathy Smith</td> <td>Massively parallel minigene assays identify splice disruptive missense and synonymous POU1F1 variants in pituitary hormone deficiency</td></tr>
<tr><td>A11</td> <td>Da Kuang</td> <td>Tools for strategic planning, tracking and sharing of systematic variant effect mapping studies</td></tr>
<tr><td>A12</td> <td>Daniel Gitterman</td> <td>Optimization of Saturation Genome Editing to enable large scale implementation of variant effect mapping</td></tr>
<tr><td>A13</td> <td>Daniel Munro</td> <td>DeMaSk: a deep mutational scanning substitution matrix and its use for variant impact prediction</td></tr>
<tr><td>B1</td> <td>Daniel Zimmerman</td> <td>Measuring the Functional Impact of (Almost) All Possible STK11/LKB1 Missense Variants in Monogenic Disease and Cancer</td></tr>
<tr><td>B2</td> <td>David Ross</td> <td>Massively Multiplexed Measurement of Genotype-Phenotype Relationships for a Biological Sense-Response System</td></tr>
<tr><td>B3</td> <td>Deanna Church</td> <td>Digital Genome Engineering: Unlocking the Unlimited Potential of Biology</td></tr>
<tr><td>B4</td> <td>Elizabeth Morton</td> <td>An approach for directed in vivo mutagenesis in C. elegans</td></tr>
<tr><td>B5</td> <td>Florence Chardon</td> <td>A single-cell genomics approach to characterize global transcriptional consequences of genetic perturbations at scale</td></tr>
<tr><td>B6</td> <td>Gemma Montalban</td> <td>Massively parallel functional analysis of missense variants in the breast/ovarian cancer gene RAD51C</td></tr>
<tr><td>B7</td> <td>Hannah De Jong</td> <td>Characterizing hypertrophic cardiomyopathy genetic variation by high-throughput mutagenesis and cellular phenotyping</td></tr>
<tr><td>B8</td> <td>Henrike Heyne</td> <td>Predicting Functional Effects of Missense Variants in Voltage-Gated Sodium and Calcium Channels</td></tr>
<tr><td>B9</td> <td>Jackson Tonnies</td> <td>Fine scale characterization of light-responsive plant enhancers</td></tr>
<tr><td>B10</td> <td>James McCormick</td> <td>Structurally Distributed Surface Sites Tune Allosteric Regulation</td></tr>
<tr><td>B11</td> <td>James Shepherdson</td> <td>Deep Mutational Scanning of the Retinal Transcription Factor CRX</td></tr>
<tr><td>B12</td> <td>Jeannette Tenthorey</td> <td>Mutational resilience of antiviral restriction favors primate TRIM5α in host-virus evolutionary arms races</td></tr>
<tr><td>B13</td> <td>Jochen Weile</td> <td>TileSeqPro 1.0: A quality control and data processing pipeline for variant effect mapping via TileSeq</td></tr>
<tr><td>C1</td> <td>Josh Tycko</td> <td>High-Throughput Discovery and Characterization of Human Transcriptional Effectors</td></tr>
<tr><td>C2</td> <td>Juannan Zhou</td> <td>Empirical variance component regression for sequence-function relationships</td></tr>
<tr><td>C3</td> <td>Kate Crawford</td> <td>Complete mapping of the functional and antigenic effects of mutations to Lassa virus glycoprotein</td></tr>
<tr><td>C4</td> <td>Larissa Milano</td> <td>Saturation mutagenesis of the PALB2 tumor suppressor gene</td></tr>
<tr><td>C5</td> <td>Laura Haynes</td> <td>High-throughput quantitative mapping of a serine protease inhibitor’s mutational landscape</td></tr>
<tr><td>C6</td> <td>Mafalda Dias</td> <td>Large-scale clinical interpretation of genetic variants using evolutionary data and deep learning</td></tr>
<tr><td>C7</td> <td>Malin Andersson</td> <td>Transforming diagnostic confidence in neurodevelopmental disorders using CRISPR-based saturation genome editing</td></tr>
<tr><td>C8</td> <td>Matteo Cagiada</td> <td>Understanding the origins of loss of protein function by analyzing the effects of thousands of variants on activity and abundance</td></tr>
<tr><td>C9</td> <td>Mireia Seuma</td> <td>The genetic landscape for amyloid beta fibril nucleation accurately discriminates familial Alzheimer's disease mutations</td></tr>
<tr><td>C10</td> <td>Nicholas Popp</td> <td>A novel method for using Deep Mutational Scanning to study the effects of variation in extracellular proteins</td></tr>
<tr><td>C11</td> <td>Nicole Thadani</td> <td>Using coronavirus sequences and mutation effects data to predict evolution of SARS-CoV-2</td></tr>
<tr><td>C12</td> <td>Renee Geck</td> <td>Functional interpretation of human G6PD variants using multiplexed analyses in S. cerevisiae</td></tr>
<tr><td>C13</td> <td>Sayeh Gorjifard</td> <td>Dominant negative peptides of S pyogenes Cas9 as novel class of anti-CRISPRs</td></tr>
<tr><td>D1</td> <td>Shawn Fayer</td> <td>Closing the gap: systematic clinical integration of multiplexed functional data to resolve variants of uncertain significance</td></tr>
<tr><td>D2</td> <td>Sriram Pendyala</td> <td>Using Microscope-based Screening to Study Genetic Causes of Cardiomyopathy</td></tr>
<tr><td>D3</td> <td>Steven Erwood</td> <td>Adapting CRISPR prime editing for saturational variant interpretation</td></tr>
<tr><td>D4</td> <td>Taylor Sparring</td> <td>Deep mutational scan of the VWF C domains to define mutations associated with VWD</td></tr>
<tr><td>D5</td> <td>Thuy Nguyen</td> <td>Predicting allele specific epistasis in an enzyme pair</td></tr>
<tr><td>D6</td> <td>Vahid Aslanzadeh</td> <td>Massively parallel functional scanning of extracellular human insulin receptor variants</td></tr>
<tr><td>D7</td> <td>Vanessa Nguyen</td> <td>Molecular determinants of Hsp90 recognition of Src kinase revealed by deep mutational scanning</td></tr>
<tr><td>D8</td> <td>Warren van Loggerenberg</td> <td>Systematic proactive variant effect maps for HMBS</td></tr>
<tr><td>D9</td> <td>Willow Coyote-Maestas</td> <td>Insertional and mutational scanning of a potassium channel</td></tr>
<tr><td>D10</td> <td>Xiaoyan 'Isaac' Jia</td> <td>Massively parallel functional testing of missense variants in the tumor suppressor MSH2</td></tr>
<tr><td>D11</td> <td>Jenny</td> <td>?</td></tr>
</table>
</div>
</body>
<html>
